collectors/health.go

```diff
@@ -448,6 +543,117 @@ func (c *ClusterHealthCollector) cephUsageCommand() []byte {
 	return cmd
 }
 
+func (c *ClusterHealthCollector) collectRecoveryClientIO() error {
+	cmd := c.cephPlainUsage()
+	buf, _, err := c.conn.MonCommand(cmd)
+	if err != nil {
+		return err
+	}
+
+	var matched []string
+	sc := bufio.NewScanner(bytes.NewReader(buf))
+	for sc.Scan() {
+		line := strings.TrimSpace(sc.Text())
+
+		switch {
```

Comment:
It'd be nice to break out methods for gathering recovery IO and client IO.  This is pretty deeply-nested. (`gocyclo` would probably complain.)